plugins/chat/spec/models/category_channel_spec.rb

```diff
@@ -124,6 +124,13 @@
           channel.validate
           expect(channel.errors.full_messages).to include("Slug is invalid")
         end
+
+        it "does not add a validation error when autogenerated slug ends up empty" do
+          SiteSetting.slug_generation_method = "none"
+          channel.update!(name: "は無効です", slug: nil)
+          expect(channel.errors.full_messages).to be_empty
```

Comment:
Could potentially be `expect(channel).to be_valid`.

Reply:
Ah yes that's better